addon/components/input-mask.js

```diff
@@ -59,6 +59,12 @@ export default Ember.TextField.extend({
   }),
 
   setMask: function() {
+    Inputmask.extendDefinitions({
```

Comment:
Is there a better place to put this definition?  Maybe in a config file?  If @pzuraq or any other maintainer wants, I can move this elsewhere.  I'll leave it to you.